src/day7.scala

```diff
@@ -0,0 +1,76 @@
+// using resourceDir "../input"
+// using lib com.lihaoyi::pprint:0.6.6
+
+package day7
+
+import scala.util.Using
+import scala.io.Source
+import scala.annotation.tailrec
+
+extension (s: String)
+  def input: List[Int] =
+    Using.resource(Source.fromFile("input/day7"))(_.mkString).split(",").map(_.toInt).toList
```

Comment:
It seems that `s` is not used and so the extension is not really useful

Reply:
ach damn, I was converting my solution on the fly based on what I saw in @julienrf 's one

Reply:
Fixed!